metabrainz/model/payment.py

```diff
@@ -333,9 +333,13 @@ def log_stripe_charge(cls, charge):
             address_country=charge.source.address_country,
         )
 
-        if charge.metadata.is_donation is True or charge.metadata.is_donation == "True":
-            new_donation.can_contact = charge.metadata.can_contact == u'True'
-            new_donation.anonymous = charge.metadata.anonymous == u'True'
+        if charge.metadata.is_donation == "True":
+            charge.metadata.is_donation = True
+        if charge.metadata.is_donation == "False":
+            charge.metadata.is_donation = False
+        if charge.metadata.is_donation is True:
```

Comment:
could have just used `if charge.metadata.is_donation` too

Reply:
Yeah, not my code. :)